calculator/interfaces/Main.java

```diff
@@ -0,0 +1,141 @@
+package calculator.interfaces;
+import java.util.Scanner;
+import java.util.InputMismatchException;
+import java.util.regex.*;
+import java.time.ZoneId;
+
+public class Main {
+
+    public static final String basicCalculator = "1";
```

Comment:
public static final variables are names in convention like BASIC_CALCULATOR_ID

Reply:
fix them all plx

Reply:
fixed